geonode/base/migrations/0034_resourcebase_bbox.py

```diff
@@ -0,0 +1,19 @@
+# Migrating a new PolygonField (djnago GEOSGeometry) 'bbox'
```

Comment:
Probably a good idea - you would have to migrate backwards to `0033_whatever_the_name_was.py`, then delete both `0034` and `0035` and run `makemigrations` again

Reply:
Yes. But shouldn't I write a migration file of some sort similar to 0034 for the users who already have some layers in their database? So that it can add the bbox_polygon field in their database? Also, we manually configured our db to talk to postgis (DATABASE_URL=postgres://geonode:geonode@localhost:5432/geonode, etc), should we do something about that in the code as well?